control

```diff
@@ -0,0 +1,14 @@
+Package: ni-system-monitor-veristand-{veristand_version}-support
+Version: {nipkg_version}
+Architecture: windows_x64
+Maintainer: National Instruments <support@ni.com>
+XB-Plugin: file
+Description: Provides support for the system monitor custom device for NI VeriStand {veristand_version}.
+XB-Eula: eula-ni-standard
+Priority: standard
+Homepage: http://www.ni.com
+XB-LanguageSupport: en
+XB-UserVisible: yes
+Section: Add-Ons
+XB-DisplayVersion: {display_version}
+XB-DisplayName: System Monitor for VeriStand {veristand_version}
```

Comment:
By convention, the control file should have a blank line at the end of the file.